package.json

```diff
@@ -20,13 +20,17 @@
     "mime": "^1.3.4"
   },
   "engines": {
-    "node": "^10.0.0"
+    "node": "^10.0.0 || ^12.0.0 || ^14.0.0"
   },
   "repository": {
     "type": "git",
     "url": "git@github.com:AppGeo/wms.git"
   },
   "devDependencies": {
-    "tape": "^4.1.0"
+    "tap-spec": "^5.0.0",
+    "tape": "^5.2.2"
+  },
+  "volta": {
+    "node": "12.20.0"
```

Comment:
Automatically switches to 12.20.0 when working in this folder. This matches the node version of Giza.